### PR TITLE
mgr/cephadm: fix errors in command <ceph orch osd> help and documentation

### PR DESCRIPTION
**Pull request by jmolmo**
`ceph orch rm --help` --->   Now it looks like:

```
 Monitor commands: 
 =================
orch osd rm <osd_id>... [--replace] [--force]                                                              Remove OSD daemons
orch osd rm status [--format {plain|json|json-pretty|yaml}]                                                Status of OSD removal operation
orch osd rm stop <osd_id>...                                                                               Cancel ongoing OSD removal operation
```

Resolves: https://tracker.ceph.com/issues/51043

Signed-off-by: Juan Miguel Olmo Martínez <jolmomar@redhat.com>

## Checklist
- [x] References tracker ticket
- [x] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/mgr/orchestrator/module.py [hunk 1]
```diff
@@ -814,24 +814,24 @@ def _daemon_add_osd(self, svc_arg: Optional[str] = None) -> HandleCommandResult:
 
     @_cli_write_command('orch osd rm')
     def _osd_rm_start(self,
-                      svc_id: List[str],
+                      osd_id: List[str],
```
Comment (adk3798) [inline src/pybind/mgr/orchestrator/module.py:817]: Nit, but can we pluralize this parameter so it's "osd_ids" (here and other places it's being used)? I think having <osd_id> in the help output gives the impression you can only provide one. "osd_ids" also matches with what we call the argument in https://github.com/ceph/ceph/blob/master/src/pybind/mgr/cephadm/module.py#L2496

**Review by adk3798 — APPROVED** [review]

**Comment by tchaikov** [comment]
jenkins test make check

File: src/pybind/mgr/orchestrator/module.py (hunk 1 above)
Comment (jmolmo) [reply src/pybind/mgr/orchestrator/module.py:817]: mmm. not sure of this is a good idea. I am using the general rule for list parameters. The help for this command is generated automatically.. an it seems that if your a re using a list of parameters, the help automatically adds "..." like in:
`orch osd rm <osd_id>... [--replace] [--force]                  Remove OSD daemons`

So if we pluralize...maybe we need to review all other list parameters of all commands to follow the same rule... or change the command help generation.

Comment (sebastian-philipp) [reply src/pybind/mgr/orchestrator/module.py:817]: right. `orch osd rm --osd-id 1 --osd-id 2 --osd-id 3 ...` I think this is ok

**Review by sebastian-philipp — APPROVED** [review]

File: src/pybind/mgr/orchestrator/module.py (hunk 1 above)
Comment (adk3798) [reply src/pybind/mgr/orchestrator/module.py:817]: okay, that makes sense

**Comment by sebastian-philipp** [comment]
https://pulpito.ceph.com/swagner-2021-06-11_08:42:41-rados:cephadm-wip-swagner-testing-2021-06-10-1331-distro-basic-smithi/

**Pull request merged**